package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "flagparser",
+  "version": "1.0.0",
+  "description": "We are building a feature-rich flag parser.",
+  "main": "index.js",
```

Comment:
`main` should be referred from the tsConfig `outDir` key, hence it should be `build/index.tsx`

Reply:
main should be `build/index.js`